frontend/src/pages/EmptyArchivePage.tsx

```diff
@@ -0,0 +1,37 @@
+import { Stack, Typography } from '@mui/material';
+
+const EmptyArchivePage = () => {
+  return (
+    <Stack direction="column">
+      <Stack
+        direction="row"
+        padding="2.5rem 2.875rem 0 1.875rem"
+        alignItems="center"
+        justifyContent="flex-start"
+      >
+        <Typography fontSize="1.625rem" fontWeight="600">
+          Archive
+        </Typography>
+      </Stack>
+      <Stack
+        direction="column"
+        justifyContent="center"
+        alignItems="center"
+        sx={{ height: '80vh', width: '100%' }}
```

Comment:
Let's remove this

Reply:
this changes the design segnificantly

Reply:
It won't because we're adding height="100%" to the column

Reply:
the remval of the height does push everything to the top instead of the center